go.sum

```diff
@@ -3,6 +3,7 @@ github.com/bmizerany/pat v0.0.0-20160217103242-c068ca2f0aac/go.mod h1:8rLXio+Wji
 github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/fsnotify/fsevents v0.1.0 h1:qeALFWR0ZE27D78Zq8mYxCF/5YrioLzrddiDz5pbR7c=
```

Comment:
Not sure why this was missed in the initial move to go modules, but I think it belongs here. /cc @pda

Reply:
Ah - I think that's a darwin-only package? Or used by puma-dev in a darwin-only context? My build etc would have only checksummed linux packages.